premerge/docs.md

```diff
@@ -0,0 +1,84 @@
+# LLVM Premerge infra - GCP runners
```

Comment:
This page isn't linked to from anywhere and a lot of it seems redundant with the other files?

Reply:
forgot to remove it

Reply:
failed to `git rm`.. thanks!